Python/Product/TestAdapter.Executor/Pytest/PytestExtensions.cs

```diff
@@ -64,13 +64,13 @@ string projectHome
             var pytestId = CreateProperCasedPytestId(sourceFullPath, projectHome, test.Id);
             var fullyQualifiedName = CreateFullyQualifiedTestNameFromId(sourceFullPath, pytestId);
             var tc = new TestCase(fullyQualifiedName, PythonConstants.PytestExecutorUri, sourceFullPath) {
-                DisplayName = test.Name,
+                DisplayName = FixupParameterSets(test.Name),
```

Comment:
Not a huge fan of calling FixupParameterSets in two spots. I think ideally we'd get displayname from FQN